src/Symfony/Component/DependencyInjection/Compiler/Compiler.php

```diff
@@ -95,6 +102,8 @@ public function addPass(CompilerPassInterface $pass, $type = PassConfig::TYPE_BE
      */
     public function addLogMessage($string)
     {
+        @trigger_error($string);
```

Comment:
hum OK, I removed the trigger_error call for now - but kept the $level arg: even unused, it make the signature compatible with future changes requiring it.